resources/uploads/upload.yml

```diff
@@ -28,6 +28,13 @@ get:
     "422":
       $ref: "responses/upload_validation_error.yml"
 
+  x-codeSamples:
```

Comment:
Will you include code samples for other languages, like the other resources have?